lib/resources/example_json/r4_examples_service_request.rb

```diff
@@ -0,0 +1,158 @@
+            "code": "664912",
+            "display": "inflammation",
+            "userSelected": true
+          },
+          {
+            "system": "http://snomed.info/sct",
+            "code": "1415005",
+            "display": "Inflammation of lymphatics"
+          }
+        ],
+        "text": "inflammation"
+      },
+      "authoredOn": "2020-03-25T00:05:30.000Z",
+      "requester": {
+        "reference": "Practitioner/1626008",
+        "display": "McCready, Tim"
+      },
+      "performer": [
+        {
+          "reference": "Practitioner/664912",
+          "display": "Dr. Spoc"
+        },
+        {
+          "reference": "Practitioner/1744062",
+          "display": "Dr. Dre"
+        }
+      ],
+      "locationReference": [
+        {
+          "reference": "Location/309348",
+          "display": "Mayo Clinic"
+        }
+      ],
+      "reasonCode": [
+        {
+          "text": "Fracture"
+        }
+      ],
+      "bodySite": [
+        {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/1028",
+              "code": "309344",
+              "display": "Arm L",
+              "userSelected": true
+            },
+            {
+              "system": "http://snomed.info/sct",
+              "code": "362728000",
+              "display": "entire right upper extremity (body structure)"
+            }
+        ],
+        "text": "Arm L"
+        }
+      ],
+      "note": [
+        {
+          "authorReference": {
+            "reference": "Practitioner/1",
+            "display": "Some, Name"
+          },
+          "time": "2014-10-10T15:25:45.000Z",
+          "text": "A comment"
+        }
+      ],
+      "text": {
+        "status": "generated",
+        "div": "&lt;div xmlns=\"http://www.w3.org/1999/xhtml\">&lt;p>&lt;b>Service Request&lt;/b>&lt;/p>"\
+            "&lt;p>&lt;b>Patient&lt;/b>: Lagunov, Sameera&lt;/p>"\
+             "&lt;p>&lt;b>Code&lt;/b>: Resuscitation Status&lt;/p>"\
+             "&lt;p>&lt;b>Occurrence&lt;/b>: Mar 24, 2020 7:05 P.M. CDT&lt;/p>&lt;p>&lt;b>Quantity&lt;/b>: 3.0 &lt;/p>"\
+             "&lt;p>&lt;b>Priority&lt;/b>: routine&lt;/p>"\
+             "&lt;p>&lt;b>As Needed&lt;/b>: inflammation&lt;/p>"\
+             "&lt;p>&lt;b>Body Site&lt;/b>:&lt;/p>&lt;ul>&lt;li>Arm L&lt;/li>&lt;/ul>&lt;p>"\
+             "&lt;b>Reason&lt;/b>:&lt;/p>&lt;ul>&lt;li>Fracture&lt;/li>&lt;/ul>"\
+             "&lt;p>&lt;b>Status&lt;/b>: active&lt;/p>"\
+             "&lt;p>&lt;b>Intent&lt;/b>: Order&lt;/p>"\
+             "&lt;p>&lt;b>Requester&lt;/b>: McCready, Tim&lt;/p>"\
+             "&lt;p>&lt;b>Order Detail&lt;/b>: 03/24/20 19:05:00, Full Resuscitation&lt;/p>"\
+      }
+    }
+
+    R4_SERVICE_REQUEST_BUNDLE ||={
+      "resourceType": "Bundle",
+      "id": "b630105a-2ec0-4877-8f8c-3b26b53cacd9",
+      "type": "searchset",
+      "total": 1,
```

Comment:
I believe support of 'total' was removed.

Reply:
+1 lets remove